### ISSUE TITLE
Stretched images

### ISSUE THREAD

**Issue by sbrl** [est31/leaftest#19]
Hello,

I've got an annoying problem whereby leaftest stretches the edges of rendered chunks:

![Selection_140](https://user-images.githubusercontent.com/9929737/54881130-99684180-4e44-11e9-8ad9-820a5ee0c6b7.png)

I'm pretty sure the issue lies in [this imagemagick call](https://github.com/est31/leaftest/blob/master/mapper.sh#L130), but the `downscale` shell function appears to be buggy too.

```
$ montage --version
Version: ImageMagick 6.9.7-4 Q16 arm 20170114 http://www.imagemagick.org
Copyright: © 1999-2017 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP 
Delegates (built-in): bzlib djvu fftw fontconfig freetype jbig jng jp2 jpeg lcms lqr ltdl lzma openexr pangocairo png tiff wmf x xml zlib
```

The above image is just an example. It gets more extreme when you zoom out:

![Selection_141](https://user-images.githubusercontent.com/9929737/54881162-054aaa00-4e45-11e9-8cee-93a6c7458a75.png)

**Comment by est31** [comment]
Hmm can't remember seeing this bug yet. I think it has something to do with tiles at that position not being available. Have you tried adjusting the limits of the map generation?

**Comment by sbrl** [comment]
Yeah, I haven't pre-generated all the terrain in the area, and I mostly use teleporting to jump around because terrain generation can be slow.

The limits I've set cover all the terrain I've generated in patches so far.

Maybe if a chunk can't be found in the source to render from, a default tile could be used instead?
